.pylintrc

```diff
@@ -73,11 +72,41 @@ disable=spelling,       # way too noisy
     no-else-return,     # relax "elif" after a clause with a return
     docstring-first-line-empty, # relax docstring style
     import-outside-toplevel,
-    bad-continuation, bad-whitespace, # differences of opinion with black
-    import-error        # overzealous with our optionals/dynamic packages
-
-
-
+    import-error,        # overzealous with our optionals/dynamic packages
+
+    # TODO: these were added in modern Pylint and we want to enable them.
+    # (If we decide any of them should be ignored, then move it above this section
+    #  and add a comment with the rationale.)
+    arguments-renamed,
+    broad-exception-raised,
```

Comment:
I don't know why this if failing now. It's supposed to be covered by `overgeneral-exceptions` at the very bottom of this diff. Lmk if folks would like me to investigate this more.

Reply:
There were a lot of bugs in old Pylint handling of scoped suppressions (e.g. sometimes module-scoped suppressions "leaked" from one module to another, etc) so it's quite possible that one of those got squashed.